### DISCUSSION TITLE
Multi Model Endpoint Tutorial - Docker Question

### DISCUSSION THREAD

**Discussion by tugisuwon86** [aws/amazon-sagemaker-examples#3470] [Q&A]
Hello

I am following https://github.com/aws/amazon-sagemaker-examples/blob/main/advanced_functionality/multi_model_sklearn_home_value/sklearn_multi_model_endpoint_home_value.ipynb

Here, it states 'In this notebook, we will be using the AWS Managed Scikit Learn image for both training and inference - this image provides native support for launching multi-model endpoints.' I am trying to run the same notebook from AWS sagemaker, but having the following problem. 

`Your Ecr Image 366743142698.dkr.ecr.ap-northeast-2.amazonaws.com/sagemaker-scikit-learn:0.20.0-cpu-py3 does not contain required com.amazonaws.sagemaker.capabilities.multi-models=true Docker label(s)`

Which docker/image should I use to complete this notebook? If I need to create my own docker, is there an example I can follow? Thanks in advance.
